charts/chronicle/values.yaml

```diff
@@ -141,8 +143,9 @@ test:
       repository: blockchaintp/chronicle-helm-api-test-amd64
       ## @md | `test.api.image.tag` | the image tag | latest |
       tag: BTP2.1.0-0.7.3
-  ## @md | `test.enabled` | true to enable test Jobs and Services | true |
-  enabled: true
+  auth:
+    enabled: true
```

Comment:
Curious why we need this extra `auth.enabled`, it doesn't suffice to see if either URI is set?